Plugins/Traccc/include/Acts/Plugins/Traccc/CellConversion.hpp

```diff
@@ -0,0 +1,96 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+// Acts include(s)
+#include "Acts/Geometry/GeometryHierarchyMap.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+
+// Plugin include(s)
+#include "Acts/Plugins/Traccc/BarcodeMap.hpp"
+#include "Acts/Plugins/Traccc/Detail/Module.hpp"
+
+// Detray include(s)
+#include "detray/core/detector.hpp"
+
+// Traccc include(s)
+#include "traccc/edm/cell.hpp"
+#include "traccc/geometry/geometry.hpp"
+#include "traccc/io/digitization_config.hpp"
+#include "traccc/io/read_geometry.hpp"
+#include "traccc/io/reader_edm.hpp"
+
+// System include(s)
+#include <cstdint>
+#include <cstdlib>
+#include <map>
+#include <memory>
+#include <tuple>
+#include <utility>
+
+namespace {
```

Comment:
Anonymous namespaces in headers work in a non obvious way. If this needs to be in the header, put it in a `detail` namespace. (The `createCellsAndModules` function could go into a `.cpp` file along with this namespace, however.)